test/step/the_app_is_running.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:black_tax_and_white_benefits/main.dart';
+
+import '../mock_posts.dart';
+
+Future<void> theAppIsRunning(WidgetTester tester) async {
+  await tester.pumpWidget(MyApp(
```

Comment:
next time we could use build.yaml to keep these in his pckage and not our repo imho